parm/archive/gdasocean.yaml.j2

```diff
@@ -3,7 +3,10 @@ gdasocean:
     name: "GDASOCEAN"
     target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gdasocean.tar"
     required:
-        {% for fhr in range(FHMIN, FHMAX + 1, FHOUT_OCN) %}
+        {% for fhr in range(FHMIN + FHOUT_OCN, FHMAX + 1, FHOUT_OCN) %}
```

Comment:
Should this just be `FHOUT_OCN` instead of `FHMIN + FHOUT_OCN`?  Typically, `FHMIN` is 0, but if it were, say, 3, would the first ocean history file really be written at 27 forecast hours?

Reply:
My understanding is that `FHOUT_OCN` is the step (now 3 hours), so that for `DOIAU="NO",` the file at `FHMIN` hours is skipped. So if `FHMIN` were 3, the first ocean file would be at 6 hours.